### PR TITLE
9093 order style changes

### PR DESCRIPTION
**Pull request by TomElliottFlexion**
New order format: https://app.zenhub.com/files/152320868/91e61f7c-fc89-4fa8-90ae-ff52b0846c6f/download

Notice with updated header (ad hoc generated): https://app.zenhub.com/files/152320868/69ce2bf9-5af7-42d2-9c2a-6007e327bf48/download

this PR also includes some devex work we did along the way once we noticed the index.scss was a giant file trying to style every PDF.  we split the index.scss into multiple sub scss to make each PDF styled individually and easier to manage (reduce blast radius when changing pdf styles)

### CODE REVIEW

File: shared/src/business/utilities/pdfGenerator/components/PageMetaHeaderDocket.jsx [hunk 1]
```diff
@@ -6,19 +6,19 @@ export const PageMetaHeaderDocket = ({ docketNumber }) => {
       <div
         style={{
           float: 'left',
-          fontFamily: "'nimbus_roman', serif",
+          fontFamily: "'Century Schoolbook Std', serif",
           fontSize: '12px',
-          transform: 'scale(0.75)',
+          // transform: 'scale(0.75)',
```
Comment (rachelschneiderman) [inline shared/src/business/utilities/pdfGenerator/components/PageMetaHeaderDocket.jsx]: Are these needed anymore?

**Review by rachelschneiderman — APPROVED** [review]

File: shared/src/business/utilities/pdfGenerator/components/PageMetaHeaderDocket.jsx (hunk 1 above)
Comment (halprin) [reply shared/src/business/utilities/pdfGenerator/components/PageMetaHeaderDocket.jsx]: Yeah, let's remove old code if it isn't needed.

File: shared/src/business/utilities/pdfGenerator/documentTemplates/Order.jsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-      <DocketHeader
+    <div className="order-pdf">
+      <OrderPrimaryHeader />
+      <OrderDocketHeader
         caseCaptionExtension={options.caseCaptionExtension}
-        caseTitle={options.caseTitle}
+        caseTitle={options.caseTitle.toUpperCase()}
         docketNumberWithSuffix={options.docketNumberWithSuffix}
-        h3={orderTitle}
+        orderTitle={orderTitle}
```
Comment (halprin) [inline shared/src/business/utilities/pdfGenerator/documentTemplates/Order.jsx]: I love the rename of this argument!

File: shared/src/business/utilities/pdfGenerator/documentTemplates/Order.jsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ export const Order = ({ options, orderContent, orderTitle, signatureText }) => {
             Clerk of the Court
           </p>
         </div>
-      )}
-    </>
+      )}{' '}
```
Comment (halprin) [inline shared/src/business/utilities/pdfGenerator/documentTemplates/Order.jsx]: Does this just add a space at the end?  Is it needed?

File: shared/src/business/utilities/pdfGenerator/components/PageMetaHeaderDocket.jsx (hunk 1 above)
Comment (TomElliottFlexion) [reply shared/src/business/utilities/pdfGenerator/components/PageMetaHeaderDocket.jsx]: Removed. @codyseibert  (?)

**Review by halprin — APPROVED** [review]
Thanks for walking me through the code!  Approved, but I'm curious if that extra space is needed.

File: shared/src/business/utilities/pdfGenerator/documentTemplates/Order.jsx (hunk 3 above)
Comment (codyseibert) [reply shared/src/business/utilities/pdfGenerator/documentTemplates/Order.jsx]: it doesn't seem like we needed this. I'll remove it in the new pr

**Pull request closed without merging**